lib/cms.js

```diff
@@ -0,0 +1,46 @@
+import fs from 'fs'
```

Comment:
is this file auto generated?

Reply:
Nope! We can do what we like in there...

Reply:
ah ok, it looked all so well commented and stuff that I assumed it was generated! :D 
Good job for documenting it so well then!